### ISSUE TITLE
why Tensorflow 1.7 has a bad performance

### ISSUE THREAD

**Issue by xywlpo** [matterport/Mask_RCNN#1612]
Hi, I have use MaskRCNN to train my own dataset with tensorflow 1.4.0, but when I use multi-gpus to train, the program will be broken down. So I upgrade my tensorflow to version 1.7.0, the problem is solved. 
But the model which was trained with my dataset has a very bad performance, the segment mask is smaller than the object region. And when I use tensorflow 1.4.0 to train my dataset, the performance is very good. And the training code is the same. Is anyone met this problem? Why does this situation happen?

Thanks very much!!
